### DISCUSSION TITLE
The remote server returned an error: (500) Internal Server Error

### DISCUSSION THREAD

**Discussion by RoyalWulf** [nbrown02/FlowViz#130] [Q&A] [answered]
This is the first time i have used the latest release.
When I open the pbit it refreshes the data.
We have a large organisation and it takes ages.
I have changed the ChangedBy date to 2024-06-01 just to get it going
I get this error when refreshing WorkItems FlowEfficiency

![image](https://github.com/user-attachments/assets/7f30d1e9-0b5b-4826-b8ce-0296fb56588f)

it is on the Sorted Rows step
![image](https://github.com/user-attachments/assets/bcbd59d0-e1b5-4b98-96c6-9096851ec940)

The error comes up immediately.
Any thoughts on why and how to fix?
Thanks

**Comment by nbrown02** [comment, marked as answer]
Hi there,

This is because this table contains any revisions to a work item which, when working with a large project is going to be a significant load. You could try one of the unsupported templates that contains a team name parameter to filter https://github.com/nbrown02/FlowViz/raw/main/Screenshots/FlowViz%20(Project%20-%20Team%20filter).pbit

**Comment by RoyalWulf** [comment]
thanks
